modules/prediction/common/prediction_system_gflags.h

```diff
@@ -32,6 +32,12 @@ DECLARE_double(prediction_test_duration);
 
 DECLARE_string(prediction_offline_bags);
 DECLARE_int32(prediction_offline_mode);
+constexpr int kPredOnlineMode = 0;
+constexpr int kPredDumpFeatureProto = 1;
+constexpr int kPredDumpDataForLearn = 2;
+constexpr int kPredDumpPredictedTraj = 3;
+constexpr int kPredDumpFrameEnv = 4;
+constexpr int kPredDumpDataForTuning = 5;
```

Comment:
Avoid global var in header file. It will cause multi-definition error when included by multi source files and linked together.